packit_service/models.py

```diff
@@ -761,3 +771,48 @@ def get_by_pipeline_id(cls, pipeline_id: str) -> Optional["TFTTestRunModel"]:
                 .filter_by(pipeline_id=pipeline_id)
                 .first()
             )
+
+
+class InstallationModel(Base):
+    __tablename__ = "github_installations"
+    installation_id = Column(Integer, primary_key=True)
+    account_login = Column(String)
+    account_id = Column(Integer)
+    account_url = Column(String)
+    account_type = Column(String)
```

Comment:
these 4 are related to the person who installed the app, right? if so, can you make a comment about this info here please?

Reply:
added